src/math/ldexp.c

```diff
@@ -0,0 +1,18 @@
+/* This is free and unencumbered software released into the public domain. */
+
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+/**
+ * @date   2017-10-04
+ * @author Samuel Sarle
+ * @see    http://libc11.org/math/ldexp.html
+ */
+
+#include <math.h>     //For pow()
```

Comment:
Going forward have all preprocessor directives (in particular, `#include` directives) at the top of the file, following after the `#ifdef` block.